.github/workflows/deploy.yml

```diff
@@ -2,49 +2,17 @@ on:
   push:
     branches:
       - main
-
 jobs:
   deploy:
     runs-on: ubuntu-latest
-    name: Deploy to Github Pages
+    name: Deploy to Netlify
     steps:
-      - name: Checkout template repo
-        uses: actions/checkout@v3
-        with:
-          submodules: true  # Fetch Hugo themes (true OR recursive)
-          fetch-depth: 0    # Fetch all history for .GitInfo and .Lastmod
-          repository: snowplow-incubator/accelerator-web-ui-template
-
-      - name: Checkout
-        uses: actions/checkout@v3
-        with:
-          path: main
-
-      - name: Move the content folder
-        run: |
-          mv ./main/content .
-      - name: Replace configuration placeholders
-        env:
-          TITLE: Advanced Analytics for Web
-          DESCRIPTION: Tutorial for setting up advanced analytics on a website
-          BASE_URL: https://docs.snowplow.io/advanced-analytics-web-accelerator/
-          AUTHOR: Agnes Kiss & Jack Keene 
-          LANDING_PAGE_URL: /advanced-analytics-web-accelerator/en/introduction
-        run: |
-          sed -i 's|ACCELERATOR_TITLE|${{ env.TITLE }}|g' config.toml
-          sed -i 's|ACCELERATOR_DESCRIPTION|${{ env.DESCRIPTION }}|g' config.toml
-          sed -i 's|ACCELERATOR_BASE_URL|${{ env.BASE_URL }}|g' config.toml
-          sed -i 's|ACCELERATOR_AUTHOR|${{ env.AUTHOR }}|g' config.toml
-          sed -i 's|ACCELERATOR_LANDING_PAGE_URL|${{ env.LANDING_PAGE_URL }}|g' config.toml
       - name: Setup Hugo
         uses: peaceiris/actions-hugo@v2
         with:
           hugo-version: 'latest'
-          # extended: true
-
       - name: Build
-        run: hugo
-
+        run: ./.github/scripts/build.sh
```

Comment:
Can we move this out of the `.github` folder? It's not really `.github` specific now. I'd suggest a root level `.scripts` folder.